### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by yarqian** [JWSoh/MZSR#1]
HI, it is a nice job, do you have a plan to share your training code?

**Comment by JWSoh** [comment]
Yes, I am planning to release the training codes but not very soon.
